etcd/etcd-live-cluster-reconfiguration.md

```diff
@@ -229,25 +221,27 @@ member e6c2bda2aa1f2dcf is healthy: got healthy result from http://10.0.1.2:2379
 cluster is healthy
 ```
 
-If output messages don't contain any error you have to remove `/run/systemd/system/etcd2.service.d/98-force-new-cluster.conf` drop-ip and reload systemd daemon: `sudo systemctl daemon-reload` (`etcd2` daemon restart is not necessary).
+If the output contains no errors, remove the `/run/systemd/system/etcd2.service.d/98-force-new-cluster.conf` drop-in file, and reload systemd services: `sudo systemctl daemon-reload`. It is not necessary to restart the `etcd2` service after this step.
+
+The next steps are those described in the [Change the etcd Cluster Size][change-cluster-size] section, with one difference: Remove the `/var/lib/etcd2/member` directory as well as `/var/lib/etcd2/proxy`.
 
-Next steps are similar to the steps described in the [Change the etcd Cluster Size][change-cluster-size] section but with one exception: you have to remove `/var/lib/etcd2/member` directory as well as `/var/lib/etcd2/proxy` one. **NOTE**: It is strongly recommended to make a backup before you remove these directories:
+It is strongly recommended to make a backup before you remove these two directories:
 
 ```sh
 $ sudo etcdctl backup --data-dir /var/lib/etcd2 --backup-dir /var/lib/etcd2_backup
```

Comment:
Yep, good catch. I think we remove the second one, because if you've made it through creating a one-node cluster, you don't really need to do the backup yet again.